mis_builder_cash_flow/data/mis_report.xml

```diff
@@ -0,0 +1,120 @@
+  </record>
+
+  <record id="mis_kpi_acum_forecast" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">acum_forecast</field>
+    <field name="description">Acum forecast</field>
+    <field name="style_id" ref="mis_style_hidden"/>
+    <field name="sequence">15</field>
+    <field name="expression">bale[][('line_type', '=', 'forecast_line')]</field>
+  </record>
+
+  <record id="mis_kpi_aliquidity" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">liquidity</field>
+    <field name="description">LIQUIDITY</field>
+    <field name="style_id" ref="mis_style_account_sub_total"/>
+    <field name="auto_expand_accounts" eval="True"/>
+    <field name="auto_expand_accounts_style_id" ref="mis_style_account_detail"/>
+    <field name="sequence">20</field>
+    <field name="expression">bal[][('account_id.internal_type', '=', 'liquidity'), ('line_type', '=', 'move_line')]</field>
+  </record>
+
+  <record id="mis_kpi_in_total" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">in_total</field>
+    <field name="description">IN TOTAL</field>
+    <field name="style_id" ref="mis_style_account_sub_total"/>
+    <field name="sequence">30</field>
+    <field name="expression">in_receivable + in_forecast</field>
+  </record>
+
+  <record id="mis_kpi_in_receivable" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">in_receivable</field>
+    <field name="description">In receivable</field>
+    <field name="style_id" ref="mis_style_account_line"/>
+    <field name="auto_expand_accounts" eval="True"/>
+    <field name="auto_expand_accounts_style_id" ref="mis_style_account_detail"/>
+    <field name="sequence">50</field>
+    <field name="expression">bal[][('account_id.internal_type', '=', 'receivable'), ('full_reconcile_id', '=', False), ('line_type', '=', 'move_line')]</field>
+  </record>
+
+  <record id="mis_kpi_in_forecast" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">in_forecast</field>
+    <field name="description">In forecast</field>
+    <field name="style_id" ref="mis_style_account_line"/>
+    <field name="auto_expand_accounts" eval="True"/>
+    <field name="auto_expand_accounts_style_id" ref="mis_style_account_detail"/>
+    <field name="sequence">70</field>
+    <field name="expression">bal[][('line_type', '=', 'forecast_line'), ('debit', '!=', 0.0)]</field>
+  </record>
+
+  <record id="mis_kpi_out_total" model="mis.report.kpi">
+    <field name="report_id" ref="mis_report_cash_flow"/>
+    <field name="name">out_total</field>
+    <field name="description">OUT TOTAL</field>
```

Comment:
I'm not sure about this changes because, if I'm not wrong, if we leave the TOTAL inside parenthesis, it means that we can avoid reading it, we would have many "OUT" and many "IN"

I still think this can be improved but I'm not an english expert. Perhaps "Total/Receivable/Forecast In/Out?" Or "Total/Receivable/Forecast Inbound/Outbound?"

Reply:
I think it is more about finding the usual structure.

What about something like 
RECEIPTS-Total
RECEIPTS-Receivable
RECEIPTS-Forecast

and 

PAYMENTS-Total
PAYMENTS-Payable
PAYMENTS-Forecast

I just took an example [here](https://www.nab.com.au/business/moments/manage/cash-flow-tax/cash-forecast-template)
![image](https://user-images.githubusercontent.com/7600613/51783168-56386d80-2136-11e9-8ccd-c5c29d60ee03.png)